docs/src/main/sphinx/sql/create-view.rst

```diff
@@ -48,6 +49,12 @@ Create a simple view ``test`` over the ``orders`` table::
     SELECT orderkey, orderstatus, totalprice / 2 AS half
     FROM orders
 
+Create a view ``test_with_comment`` with a view comment::
+
+    CREATE VIEW test_with_comment COMMENT 'A view to keep track of orders.' AS
```

Comment:
```
CREATE VIEW test_with_comment 
    COMMENT 'A view to keep track of orders.' 
AS
    SELECT orderkey, orderstatus, totalprice
    FROM orders
```

Reply:
I would add the indents as well ..

Reply:
maybe with only two spaces..

Reply:
While I like adding indents, there's no such consensus and `CTAS` example have no indents.

Reply:
CREATE MATERIALIZED VIEW does have the indents .. and CTAS and others should have as well. We are aiming to make it as readable as possible and attempt to follow the Mozilla guidelines - https://docs.telemetry.mozilla.org/concepts/sql_style.html

Reply:
> We are aiming to make it as readable as possible and attempt to follow the Mozilla guidelines - https://docs.telemetry.mozilla.org/concepts/sql_style.html

Please document in DEVELOPMENT.md or somewhere.

Reply:
I would merge as-is since indents only for comment example doesn't make sense.